### ISSUE TITLE
Form view change (Terri's suggestion)

### ISSUE THREAD

**Issue by blaughman** [usdoj-crt/crt-portal-management#1389]
## User story:
As a visually impaired CRT user, I need "show up to" removed as a selectable option from the form view change dropdown box.

## Objective:
Improve accessibility of the site.

## Ticket background:
After testing with our accessibility specialist, the screen reader was capturing "show up to" as a selectable option.  This is not the desired for a visually impaired user.  "Show up to" should be a label, not a menu option.
